tools/blocksconvert/plan_file.go

```diff
@@ -19,15 +19,15 @@ import (
 
 type PlanEntry struct {
 	// Header
-	User     string `json:"user"`
-	DayIndex int    `json:"day_index"`
+	User     string `json:"user,omitempty"`
```

Comment:
Has the `omitempty` been added to avoid empty JSON properties when it gets serialised? Or is there a different reason?

Reply:
That exactly is the reason. On manual inspection of some plan files I've realized that all fields are *always* included, which is wasteful and makes plan files bigger than necessary.